src/utils/setup/article-rss-feed.js

```diff
@@ -0,0 +1,14 @@
+const { rssFeedArticleData } = require('../../queries/rss-feed-article-data');
+const { serializeRssData } = require('./serialize-rss-data');
+
+const articleRssFeed = {
+    serialize: serializeRssData,
+    query: rssFeedArticleData,
+    output: '/rss.xml',
+    title: 'MongoDB Developer Hub RSS Feed',
+    image_url: 'https://media.mongodb.org/favicon.ico',
```

Comment:
This might actually need to be a bigger image for some readers, but it'll work for now. Couldn't find a good reference for sizes on this online